packages/vm/README.md

```diff
@@ -14,14 +14,16 @@ Implements Ethereum's VM in Javascript.
 
 The VM currently supports the following hardfork rules:
 
+- `Chainstart` (a.k.a. Frontier) (v5, UNRELEASED)
+- `Homestead` (v5, UNRELEASED)
+- `TangerineWhistle` (v5, UNRELEASED)
+- `SpuriousDragon` (v5, UNRELEASED)
```

Comment:
Let's not change the READMEs for now on these PRs, then we can actually relatively gently move forward and merge these PRs independently on the way towards a release.

Reply:
For now I have added some "v5, UNRELEASED" notes here.